crates/swc_ecma_lints/src/config.rs

```diff
@@ -0,0 +1,22 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Clone, Copy, Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+pub enum LintRuleReaction {
+    Off,
+    Warning,
+    Error,
+}
+
+impl Default for LintRuleReaction {
+    fn default() -> Self {
+        Self::Off
+    }
+}
+
+#[derive(Debug, Clone, Default, Copy, Serialize, Deserialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct LintConfig {
+    pub no_console: LintRuleReaction,
```

Comment:
Can you add `#[serde(default)]` here?

Reply:
@kdy1 ok.

Now I think about many rules that have settings e.g. no-console has exceptions for methods seems like

```
"no-console": ["error", { "allow": ["warn"] }]
```

maybe we're should use similar structure? 

```
struct RuleConfig<T: Serialize + Deserialize + rest...>(LintRuleReaction, T);

struct NoConsoleConfig {
  allow: Optional<HashSet<String>>,
}

struct LintConfig {
  no_console: LintRuleReaction<NoConsoleConfig>
}
```

Reply:
Yeah, I think it's the correct approach, and we need to accept various values like `"error"` or `{ "allow": "warn" }`

But I think we can postpone it at the moment.

Reply:
Ok. I'll fix now

Reply:
Oh, cool!
Thanks